beps/0001-notifications-system/README.md

```diff
@@ -178,6 +178,25 @@ Let's walk through the process of sending of a notification to a user:
 
    7b. Rather than rendering the notification directly, the notifications frontend plugin leaves that to an extension installed by the frontend part of the backend plugin that sent the notification. This allows for a more customized notification experience which can also contain custom actions and data display. This step is not in scope for this proposal.
 
+### Integration with external systems
+
+The `notifications-backend` is not meant to expose the API outside of the Backstage, only other Backstage backend services are expected to connect to through the NotificationsService.
+
+To allow external system to send messages to either individual users or broadcast, a plugin `notifications-external-backend` will be created.
+
+The purpose of the plugin is to expose new REST API for creating messages, so far no reading functionality is expected. Internally, the new API will perform
+
+- external service authentication (aligned with capabilities of the Auth service, so far just shared-secret)
```

Comment:
@drodil @freben What are your suggestions here, please? The old auth service supported shared secrets only for external callers. Has this been changed recently, please?

If not, I would keep a list of shared secrets, one per external system.

Reply:
See this for external tokens: https://github.com/backstage/backstage/blob/master/packages/backend-plugin-api/src/services/definitions/AuthService.ts#L39